corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -21,9 +21,15 @@ class Command(BaseCommand):
         Pass either a domain name (to migrate all apps in the domain) or an
         individual app id. Will skip any apps that have already been migrated.
     '''
+    option_list = (
+        make_option('--usercase',
```

Comment:
I think this style of argument is deprecated in django(not sure what version) you'll want to use `add_arguments` https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/